### DISCUSSION TITLE
drop models command

### DISCUSSION THREAD

**Discussion by CTW01218** [dbt-labs/dbt-core#10169] [Ideas]
When using a dbt project repo coupled with a ci/cd proccess, sometimes there is the need to drop models and for that one needs to delete the model in the cod and then drop the materializations in the dwh by running sql scripts.
A dbt drop command (preferably with a confirmation mechanism) would be interesting to be added to dbt. The selection of models to be dropped could be done with the --select or using selectors as with models.

**Comment by ScottHAZI** [comment]
I need this one as well. Upvoted.
